store/storetest/preference_store.go

```diff
@@ -320,15 +320,15 @@ func testPreferenceDelete(t *testing.T, ss store.Store) {
 
 	store.Must(ss.Preference().Save(&model.Preferences{preference}))
 
-	if prefs := store.Must(ss.Preference().GetAll(preference.UserId)).(model.Preferences); len([]model.Preference(prefs)) != 1 {
+	if prefs, err := ss.Preference().GetAll(preference.UserId); len(prefs) != 1 {
```

Comment:
To be consistent with the previous version you should check the `err` here too.
I would suggest something like:
```go
	prefs, err := ss.Preference().GetAll(preference.UserId)
	require.Nil(t, err)
	assert.Len(t, prefs, 1, "should've returned 1 preference")
```